R/dvhydrograph-render.R

```diff
@@ -158,7 +164,8 @@ createDVHydrographRefPlot <- function(reportObject, series, descriptions) {
   series_number <- switch(series,
     'firstReferenceTimeSeries' = '1',
     'secondReferenceTimeSeries' = '2',
-    'thirdReferenceTimeSeries' = '3'
+    'thirdReferenceTimeSeries' = '3',
+    'thirdReferenceTimeSeries' = '4'
```

Comment:
good catch, I actually need to delete the fourth Reference TS, since there are still only 3...